### PR TITLE
Add calibration files for Left Arm

### PR DESCRIPTION
**Pull request by egordon**
Also added a README to the config folder about how to re-calibrate yourself.

And where to copy the files for default setup.

### CODE REVIEW

File: geodude_hardware/config/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+These files follow the updated instructions for vertical mounting found on [Barrett's Web Documentation](https://web.barrett.com/support/WAM_Documentation/Libbarrett_WAM_Vertical_Mount_Instructions.pdf)
+
+To update your local default calibration, copy these files into `/etc/barrett` after installing `libbarrett`.
+
+For example:
+* `calibration.conf` -> `/etc/barrett/calibration.conf`
+* `wam7w.conf` -> `/etc/barrett/wam7w.conf`
+* `left_wam7w/` -> `/etc/barrett/wam7w/`
+
+The first two configuration files should be fine for both arms, as they only contain variations for mounting vertically. But the folders are arm-specific.
```
Comment (atharva620) [inline geodude_hardware/config/README.md:13]: Elaborate more on why we need two different configuration folders for both arms

File: geodude_hardware/config/left_wam7w/gravitycal.conf [hunk 2]
```diff
@@ -0,0 +1,4 @@
+gravity_compensation :
```
Comment (atharva620) [inline geodude_hardware/config/left_wam7w/gravitycal.conf:1]: Make an issue: The results from this gravity calibration routine yield unsatisfactory results for J4 and J5. Need to add more poses especially for the above mentioned joints.

**Review by atharva620 — APPROVED** [review]
TODO: Attend to the comments before merging

File: geodude_hardware/config/left_wam7w/gravitycal.conf (hunk 2 above)
Comment (egordon) [reply geodude_hardware/config/left_wam7w/gravitycal.conf:1]: Added #4

**Pull request merged**